src/hardware/addon-protocols/rs232-usb.md

```diff
@@ -80,25 +72,24 @@ The unit refers to the unit of measurement that you wish to see in MyGeotab afte
 
 - The third-party device is sending a distance in 0.1 km increments, such that a value of 1223 represents a distance of 122.3 km. A multiplier of 0.1 should be applied so that the value is properly converted to kilometers on MyGeotab.
 
+<!-- The below note will be added once the Status ID link is available -->
 
+<!-- *Note* : There are some Status Data IDs that are restricted to coming from the Go Device, an extensive list of the Add-On device range can be found *linked here* -->
```

Comment:
Isn't this list available now?

Reply:
Indeed. To simplify the commit history, I will change this via another case https://jira.geotab.com/browse/SOLENG-2355